### ISSUE TITLE
configurator (debian): automatic DB config doesn't work

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5328]
**Describe the bug**
When you click on next step at step 2 on the configurator after you filled admin password and used automatic DB configuration, you can't continue.

Configurator asks you to fill a password for `pf` user. Root access is OK, DB is created.

**To Reproduce**
1. Go to step 2 of the configurator (on Debian)
2. Fill admin password
3. Click on next step

**Expected behavior**
Going to step 3.

**Additional context**
API calls I see when automatic DB configuration start:
- secure_installation => OK
- database/test (with username=root) => KO
- create => OK
- database/test (with username=pf, password=null and database=pf)

If I fill `pf` user password, I see an API call to `database/assign` that create `pf` user in DB.

**Comment by julsemaan** [comment]
I'm not sure I can do much for this. If the API call is firing with an empty password, we'll have to see with @cgx

**Comment by lzammit** [comment]
Same behaviour on CentOS Linux release 7.7.1908 (Core).

PacketFence build: 
````
packetfence-9.3.9-20200408101310.134003519.0007.devel.el7.x86_64
````
If Can add more debug:
![image](https://user-images.githubusercontent.com/5261214/78782670-5bb35980-7970-11ea-9653-28ab6faba814.png)

secure_installation works:

![image](https://user-images.githubusercontent.com/5261214/78782906-c369a480-7970-11ea-8f79-9b461825f7e4.png)

First test fails:

![image](https://user-images.githubusercontent.com/5261214/78782944-d7150b00-7970-11ea-83ed-3c12a1d27d77.png)

Create works:

![image](https://user-images.githubusercontent.com/5261214/78782966-e4ca9080-7970-11ea-80fd-757ab6ab6998.png)

Second test fails:

![image](https://user-images.githubusercontent.com/5261214/78783003-f14ee900-7970-11ea-84c1-e798635bd9a1.png)

**Comment by julsemaan** [comment]
Just replicated the issue and "assign" isn't being called when picking auto-configuration.

This is the sequence I see in the logs:
```
Apr  8 12:43:17 jsemaan-test pfhttpd: api-frontend-access 127.0.0.1 - - [08/Apr/2020:12:43:17 +0000] "POST /api/v1/configurator/config/base/database/secure_installation HTTP/1.1" 200 66 "https://172.20.20.98:1443/admin/alt" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:74.0) Gecko/20100101 Firefox/74.0"
Apr  8 12:43:17 jsemaan-test pfhttpd: api-frontend-access 127.0.0.1 - - [08/Apr/2020:12:43:17 +0000] "POST /api/v1/configurator/config/base/database/test HTTP/1.1" 500 80 "https://172.20.20.98:1443/admin/alt" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:74.0) Gecko/20100101 Firefox/74.0"
Apr  8 12:43:20 jsemaan-test pfhttpd: api-frontend-access 127.0.0.1 - - [08/Apr/2020:12:43:20 +0000] "POST /api/v1/configurator/config/base/database/create HTTP/1.1" 200 65 "https://172.20.20.98:1443/admin/alt" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:74.0) Gecko/20100101 Firefox/74.0"
Apr  8 12:43:20 jsemaan-test pfhttpd: api-frontend-access 127.0.0.1 - - [08/Apr/2020:12:43:20 +0000] "POST /api/v1/configurator/config/base/database/test HTTP/1.1" 500 78 "https://172.20.20.98:1443/admin/alt" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:74.0) Gecko/20100101 Firefox/74.0"
```

**Comment by cgx** [comment]
Fixed by 3190d86.
